### ISSUE TITLE
When I try to encrypt a 256 length vector, the ArrayToVector function converts the length to 0

### ISSUE THREAD

**Issue by Gmlife** [SergeyBel/AES#59]
AES.cpp:488 std::vector<unsigned char> AES::ArrayToVector(unsigned char *a, unsigned char len)
Should it be declared as ”ArrayToVector(unsigned char *a, unsigned int len)“?
If it is a mistake in writing, please modify it. If it is not to explain the design idea, thank you

**Comment by SergeyBel** [comment]
You are right: `ArrayToVector(unsigned char *a, unsigned char len)` should be declared as `ArrayToVector(unsigned char *a, unsigned int len)`. Fixed in https://github.com/SergeyBel/AES/commit/e4ea9ae315fb0d4df2acb22b1e36d1111a17118a
